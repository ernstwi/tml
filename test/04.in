init foo
init bar { x <= 10 }

foo -> bar
    guard { x - y == 5 }
    sync { press }
    reset { one, two, three }
bar -> baz
    sync { press }
    reset { four }
baz -> baz
