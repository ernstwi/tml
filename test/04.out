Semantic error: 2 initial locations
